docs/eventing/README.md

```diff
@@ -340,6 +337,12 @@ example.
 - [Install the Eventing component](#installation)
 - [Run samples](./samples/)
 
+## Sending Events: Sources vs Manual
```

Comment:
I would pull this up before the list of the event sources you can install. Maybe even up to line 102, before "Architecture", since it's something that lots of people will want to know how to do.

Reply:
Have moved up